scripts/traverse.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+const bcd = require('..');
+
+const { argv } = require('yargs').command('$0 <browser> [folder] [value]', 'Test for specified values in any specified browser', (yargs) => {
+  yargs.positional('browser', {
+    describe: 'The browser to test for',
+    type: 'string'
+  }).positional('folder', {
+    describe: 'The folder to test',
+    type: 'string',
+    default: ''
+  }).positional('value', {
+    describe: 'The value(s) to test against',
+    type: 'array',
+    default: [null, true]
+  });
+});
+
+function traverseFeatures(obj, depth, identifier) {
+  depth--;
+  if (depth >= 0) {
+    for (let i in obj) {
+      if (!!obj[i] && typeof(obj[i]) == "object" && i !== '__compat') {
+         if (obj[i].__compat) {
+           let comp = obj[i].__compat.support;
+           let browser = comp[argv.browser];
```

Comment:
Thanks for adding the `folder` argument!

If you now do `npm run traverse safari`, `browser` will be `undefined` in this line here when it hits the `webextensions/` folder, for example, and the scripts errors out. Need to catch that :)

Reply:
Good catch (pun not intended), fixing now!

Reply:
Thanks for the fix! I think this is the right thing to do, but now the output of `npm run traverse safari` is cluttered with `webextensions/` data points, which is technically correct of course, but now that I see this (sorry for not saying that in the first place!), I actually think for our use case we probably want to exclude webextension data from the "all folders" output (as we won't add more browsers to the data in `webextensions/` unless a browser decides to support this technology. Adding `false` data points is not worth it). 

So, this fix is fine and needs to stay in, but now I would also exclude the webextension folder from the "all folders" traversal. We've also excluded this in the year goal. I hope that makes sense :)

Reply:
That makes sense, yeah!  In fact, this gives me a chance to add another little feature I was thinking of, where multiple folders can be defined at once (identifiers already start with the folder name as well).  My plan is to convert `[folder]` into an array argument, and set the default to all the different categories minus `webextensions/` (so it can still be checked with `npm run traverse`, but it's not default behavior).